### DISCUSSION TITLE
大佬想问下这里报错咋办

### DISCUSSION THREAD

**Discussion by Eleina-233** [tabby-sec/tabby#24] [Q&A]
<img width="933" alt="image" src="https://user-images.githubusercontent.com/76209763/160965087-ff31a357-d16e-4db4-afc4-d51b31e9e3a1.png">
<img width="890" alt="image" src="https://user-images.githubusercontent.com/76209763/160965236-1961aaf8-327d-45c6-9c19-9c285160fae2.png">

**Comment by wh1t3p1g** [comment]
看这个错误排查不出来。
可以确认一下，本次运行之前是否删除之前的db文件

**Comment by r00t4dm** [comment]
加Xmx

**Reply by TimeSHU** [reply]
老哥，加了也没用。。还是报这个错误。。

**Reply by r00t4dm** [reply]
删除之前的db文件试试
